packages/help-extension/src/index.tsx

```diff
@@ -172,96 +172,95 @@ function activate(app: JupyterLab, mainMenu: IMainMenu, palette: ICommandPalette
     if (kernelInfoCache.has(sessionModel.kernel.name)) {
       return;
     }
-    serviceManager.sessions.connectTo(sessionModel).then(session => {
-      session.kernel.ready.then(() => {
-        // Check the cache second time so that, if two callbacks get scheduled,
-        // they don't try to add the same commands.
-        if (kernelInfoCache.has(sessionModel.kernel.name)) {
-          return;
+    const session = serviceManager.sessions.connectTo(sessionModel);
+    session.kernel.ready.then(() => {
+      // Check the cache second time so that, if two callbacks get scheduled,
+      // they don't try to add the same commands.
+      if (kernelInfoCache.has(sessionModel.kernel.name)) {
+        return;
+      }
+      // Set the Kernel Info cache.
+      const name = session.kernel.name;
+      const kernelInfo = session.kernel.info;
+      kernelInfoCache.set(name, kernelInfo);
+
+      // Utility function to check if the current widget
+      // has registered itself with the help menu.
+      const usesKernel = () => {
+        let result = false;
+        const widget = app.shell.currentWidget;
+        if (!widget) {
+          return result;
         }
-        // Set the Kernel Info cache.
-        const name = session.kernel.name;
-        const kernelInfo = session.kernel.info;
-        kernelInfoCache.set(name, kernelInfo);
-
-        // Utility function to check if the current widget
-        // has registered itself with the help menu.
-        const usesKernel = () => {
-          let result = false;
-          const widget = app.shell.currentWidget;
-          if (!widget) {
-            return result;
+        helpMenu.kernelUsers.forEach(u => {
+          if (u.tracker.has(widget) &&
+              u.getKernel(widget) &&
+              u.getKernel(widget).name === name) {
+            result = true;
           }
-          helpMenu.kernelUsers.forEach(u => {
-            if (u.tracker.has(widget) &&
-                u.getKernel(widget) &&
-                u.getKernel(widget).name === name) {
-              result = true;
-            }
+        });
+        return result;
+      };
+
+      // Add the kernel banner to the Help Menu.
+      const bannerCommand = `help-menu-${name}:banner`;
+      const spec = serviceManager.specs.kernelspecs[name];
+      const kernelName = spec.display_name;
+      let kernelIconUrl = spec.resources['logo-64x64'];
+      if (kernelIconUrl) {
+        let index = kernelIconUrl.indexOf('kernelspecs');
+        kernelIconUrl = baseUrl + kernelIconUrl.slice(index);
+      }
+      commands.addCommand(bannerCommand, {
+        label: `About the ${kernelName} Kernel`,
+        isVisible: usesKernel,
+        isEnabled: usesKernel,
+        execute: () => {
+          // Create the header of the about dialog
+          let headerLogo = (<img src={kernelIconUrl} />);
+          let title = (
+            <span className='jp-About-header'>,
```

Comment:
I think this happened when converting the dialogs to JSX, but I saw it when testing: this JSX has commas in it that it should not have.

Reply:
I went ahead and fixed the issue, hope you don't mind @jasongrout